tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -935,9 +935,11 @@ fib/test_fib.py::test_ipinip_hash:
 
 fib/test_fib.py::test_nvgre_hash:
   skip:
-    reason: 'Nvgre hash test is not fully supported on VS and Broadcom platform; Not supported on M*'
+    reason: 'Nvgre hash test is not fully supported on VS and Broadcom and SPC1 platform; Not supported on M*'
+    conditions_logical_operator: "OR"
     conditions:
       - "asic_type in ['vs', 'broadcom'] or topo_type in ['m0', 'mx', 'm1', 'm2', 'm3']"
+      - "asic_gen == 'spc1' and 't1-lag' in topo_name and https://github.com/sonic-net/sonic-mgmt/issues/17526"
```

Comment:
@congh-nvidia why only t1-lag? This should be for all SPC1 SKUs, right? Could you please clarify.

Reply:
Hi @kperumalbfn, currently for all the topologies we test for spc1, the test only fails on the t1-lag topology due to the limitation. And the reason is only in t1-lag we have 2 members in the portchannel.

Reply:
Thanks @congh-nvidia. Could you please fix the script for 2 LAG members instead of skipping it.

Reply:
@kperumalbfn Is this feature required for Mellanox-SN2700 T1? I think we can skip/xfail it if it's not required.